jekyll.yml

```diff
@@ -79,6 +79,8 @@ mesh_product_name_path: kuma
 mesh_namespace: kuma-system
 mesh_cp_name: kuma-control-plane
 mesh_cp_zone_sync_name_prefix: kuma-
+mesh_docker_org:
+mesh_download_url: 
```

Comment:
@slonka I wasn't quite sure how to write the variables here so they worked for Kong Mesh and Kuma